### PR TITLE
Add support for acceleration sensor in Samsung Smartthings Multipurpo…

### PR DESCRIPTION
**Pull request by robstonham-personal**
…se sensor (2016 model)

Add the attribute that indicates the device has been moved.

I'm not sure sure how to add this purely in Zigbee-herdsman-converters, so needed to add the following to the Zigbee-herdsman package as well.

file: src/zcl/definition/manufacturerCode.ts:
export default {
    Centralite: 0x104E,
    Philips: 0x100B,
    Sinope: 0x119C,
    Stelpro: 0x1185,
    Ubisys: 0x10f2,
    LegrandNetatmo: 0x1021,
    SmartThings: 0x110a,
};

and
file: src/zcl/definition/cluster.ts:
    manuSpecificSamsungAccelerometer: {
        ID: 0xFC02,
        manufacturerCode: ManufacturerCode.SmartThings,
        attributes: {
             motion_threshold_multiplier: {ID: 0, type: DataType.uint8},
             motion_threshold: {ID: 2, type: DataType.uint16},
             acceleration: {ID: 16, type: DataType.bitmap8},
             x_axis: {ID:18, type: DataType.int16},
             y_axis: {ID:19, type: DataType.int16},
             z_axis: {ID:20, type: DataType.int16},
        },
        commands: {},
        commandsResponse: {},
    },

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -5050,6 +5050,15 @@ const converters = {
             return {humidity: calibrateAndPrecisionRoundOptions(humidity, options, 'humidity')};
         },
     },
+
+    smartthings_acceleration: {
+        cluster: 'manuSpecificSamsungAccelerometer',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            return {acceleration: msg.data['acceleration'] === 1 ? true : false};
```
Comment (Koenkk) [inline converters/fromZigbee.js:5058]: Is this a state or an event? To me it looks like an event if that is indeed the case you can change this to `return {action: 'acceleration'};`

Comment (robstonham-personal) [reply converters/fromZigbee.js:5058]: To me it's a state. The device sends a message when I pickup the device, and then sends another message after it stops moving for a few seconds.

**Comment by dejanzelic** [comment]
+1 for support of this! I have a 2016 F-MLT-US-2 if we need to test anything

File: converters/fromZigbee.js (hunk 1 above)
Comment (Koenkk) [reply converters/fromZigbee.js:5058]: Clear! In that case `accelerating` or `moving` would be a better name

Comment (robstonham-personal) [reply converters/fromZigbee.js]: Sorry for the delay, been away.
I've updated to use "moving" as suggested

File: converters/fromZigbee.js [hunk 2]
```diff
@@ -5050,6 +5050,15 @@ const converters = {
             return {humidity: calibrateAndPrecisionRoundOptions(humidity, options, 'humidity')};
         },
     },
+
+    smartthings_acceleration: {
+        cluster: 'manuSpecificSamsungAccelerometer',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            return {acceleration: msg.data['moving'] === 1 ? true : false};
```
Comment (Koenkk) [inline converters/fromZigbee.js:5058]: I guess you meant to change this to?

```js
return {moving: msg.data['acceleration'] === 1 ? true : false};
```

Comment (robstonham-personal) [reply converters/fromZigbee.js]: yep, so sorry, fixed it as suggested

**Comment by Koenkk** [comment]
Thanks!

**Comment by dejanzelic** [comment]
@robstonham-personal were you able to confirm this as working? I have 2 Samsung Smartthings v4 multisensors that never toggle the "moving" attribute. The moving attribute is recongnised on the devices (and shows in mqtt and home assistant) but moving is never triggered to be true (no matter how much I move the sensor). I know that these sensors have the accelerometer because they used to work with ZHA.

**Comment by robstonham-personal** [comment]
Yep I had it working last time I looked at it.  After I updated though I
found that it stopped working for me too.
My logs showed this though:
Zigbee2MQTT:debug 2020-08-23 23:04:12: No converter available for
'F-MLT-US-2' with cluster 'manuSpecificSamsungAccelerometer' and type
'attributeReport' and data '{"acceleration":1}'

I just added "fz.smartthings_acceleration" to the fromZigbee section
in "node_modules/zigbee-herdsman-converters/devices.js"

**Comment by dejanzelic** [comment]
Thanks @robstonham-personal I created the following PR to fix this: https://github.com/Koenkk/zigbee-herdsman-converters/pull/1498 After making the change you suggested, it started working for me.

**Pull request merged**